erizo/src/erizo/handlers/HandlerImporter.cpp

```diff
@@ -0,0 +1,28 @@
+//
```

Comment:
nit: We don't use this comment anywhere else in the code. While we are not 100% consistent I wouldn't add a new thing